smqtk_iqr/web/__init__.py

```diff
@@ -52,7 +50,9 @@ def get_default_config(cls):
         }
 
     @classmethod
-    def from_config(cls, config_dict, merge_default=True):
+    def from_config(
```

Comment:
I'd like to get verification that this is done correctly

Reply:
You're not using `T` here that you've defined above, instead you're specifying `"SmqtkWebApp"`. There is a difference: when specifying `"SmqtkWebApp"` you are declaring that only that specific type will ever be returned here, but that is not the case for sub-classes, who will return their instance type, not their parent type. Using `T` should solve your issue because the specification of `T` above includes the set of types including `SmqtkWebApp` and all descendant class types.

Reply:
Eh, just tried the edit and it didn't remove the mypy error, so my first hunch was not true.

Reply:
Ahhh, found it: `cls` should be `Type[T]`, not just the `T` by itself. Just specifying `T` as the type of a param is interpreted as an input instance of type `T`, however `cls` is of course the type itself, the notation of which is `Type[T]`.